Mentioning the GDExtension icon support (#7461)
godotengine/godot-docs: tutorials/scripting/gdextension/gdextension_cpp_example.rst

.. image:: img/gdextension_cpp_animated.gif

Custom editor icon
------------------

> By default, Godot uses the node icon in the scene dock. The custom icon can be
> added via the ``gdextension`` file. Write a line where the name of the node
> class equals the resource path to the icon's SVG file.

Reviewer: ```suggestion
By default, Godot uses the node icon it is inheriting from in the scene dock. The custom icon can be
added via the ``gdextension`` file.  The node's icon is set by reference to its name and resource path
of an SVG file.
```

Author: Ummm my editor is not using the icon node it is inheriting from. Is it a bug?

![Bug](https://github.com/godotengine/godot-docs/assets/2855916/5e38d2c4-0e97-4ec0-ab7b-45347a04a43b)

Reviewer: In that Case I am wrong. Then leave out the inheriting part.

For example:

.. code-block:: none

    [Icon]
    GDExample = "res://icons/GDExample.svg"

> Path points to 16 by 16 pixel SVG image. Read the guide for :ref:`creating icons <doc_editor_icons>`
> for more information.

Reviewer: ```suggestion
The path should point to a 16 by 16 pixel SVG image. Read the guide for :ref:`creating icons <doc_editor_icons>`
for more information.
```
